test/CacheManager.Tests/RedisTests.cs

```diff
@@ -1408,6 +1408,152 @@ public void Redis_ValueConverter_ValidateValuesTypesNotUsingSerializer<T>(T valu
             triggerResult.Should().BeTrue("Event should get triggered through the backplane.");
             eventTriggeredLocal.Should().Be(expectedRemoteTriggers, "Local cache event should be triggered one time");
         }
+
+        [Fact]
+        public void Redis_GetAllKeys()
+        {
+            var testLogger = new TestLogger();
+            var cfg = ConfigurationBuilder.BuildConfiguration(settings =>
```

Comment:
you can use `TestManagers.CreateRedisCache` or similar to create standard caches